tfhe/src/integer/server_key/crt/add_crt.rs

```diff
@@ -57,7 +57,8 @@ impl ServerKey {
         ct_right: &CrtCiphertext,
     ) -> Result<(), CheckError> {
         for (ct_left_i, ct_right_i) in ct_left.blocks.iter().zip(ct_right.blocks.iter()) {
-            self.key.is_add_possible(ct_left_i, ct_right_i)?;
+            self.key
+                .is_add_possible(ct_left_i.noise_degree(), ct_right_i.noise_degree())?;
```

Comment:
I'm guessing that change is required somehow, to take noise degree instead of the ciphertext ? can you elaborate on why ?

Reply:
why was it not possible to use is_sub_possible in smart_sub ?